packages/neuron-wallet/tests/models/transaction-size.test.ts

```diff
@@ -16,7 +16,7 @@ describe('TransactionSize', () => {
     type: new Script('0xece45e0979030e2f8909f76258631c42333b1e906fd9701ec3600a464a90b8f6', '0x', ScriptHashType.Data),
   })
 
-  const witnessArgs = new WitnessArgs('', '0x', '')
+  const witnessArgs = new WitnessArgs(undefined, '0x', undefined)
```

Comment:
`''` should be handled too because we don't know if there are `''`s passed to this method.

Reply:
updated: https://github.com/nervosnetwork/neuron/pull/2803/commits/b50a7edaccf69264bffb28295bcf47a466822cb3